Directory.Build.targets

```diff
@@ -123,6 +123,17 @@
     <SuppressPackageTargetFrameworkCompatibility Include="$(UAPvNextTFM)" />
   </ItemGroup>
 
+  <!-- 
+    Import Test SDK props and targets.
+    This needs to happen here as the IsInvokableTestProject prop is required
+    which isn't available before Directory.Build.targets.
+    TODO: Fix VB test project reference to Test SDK.
```

Comment:
Is this TODO part of this PR?

Reply:
No that's an open issue: https://github.com/dotnet/corefx/issues/31875